scripts/license/LICENSE-header.md

```diff
@@ -0,0 +1,12 @@
+## License
+
+Mapbox Maps for iOS version __MAPS_SDK_VERSION__
+Mapbox Maps iOS SDK
+
+Copyright &copy; 2021 - 2023 Mapbox, Inc. All rights reserved.
```

Comment:
@sbma44 Would it make sense to update the end of copyright year range automatically?

Reply:
Proceeding with tokenizing the end year as per private conversation.